packages/vite/src/node/plugins/resolve.ts

```diff
@@ -50,6 +50,7 @@ export interface ResolveOptions {
   extensions?: string[]
   dedupe?: string[]
   preserveSymlinks?: boolean
+  absoluteUrls?: boolean
```

Comment:
For Svelte, I don't think this would be useful as the Svelte compiler doesn't auto transform `src` to `import`s like Vue does. Instead it has to be done separately through a preprocessor like https://github.com/bluwy/svelte-preprocess-import-assets, and a preprocessor is bundler-agnostic so we can't read from Vite config here.

I'm not sure if there's any other framework that can benefit from this (marko, astro?), but if not I think we should remove this option and keep it in plugin-vue at the meantime.